test.sh

```diff
@@ -11,4 +11,4 @@ if [ ! -d "./node_modules/seneca" ]; then
   npm install seneca@plugin
 fi
 
-./node_modules/.bin/mocha test/*.test.js
+./node_modules/.bin/mocha test/*.test.js --timeout 3000
```

Comment:
I will definitely be doing that as a next step.  Was trying to keep the tests as-is for this initial part of the rewrite to prove that I didn't break anything :)